modules/calc/src/main/java/com/opengamma/strata/calc/Column.java

```diff
@@ -34,66 +36,122 @@
 @BeanDefinition
 public final class Column implements ImmutableBean {
 
-  /** The definition of the column which specifies the column name and the measures it contains. */
+  /**
+   * The measure to be calculated.
+   * <p>
+   * This defines the calculation being performed, such as 'PresentValue' or 'ParRate'.
+   */
   @PropertyDefinition(validate = "notNull")
-  private final ColumnDefinition definition;
-
-  /** The pricing rules that apply to this column in addition to the default rules. */
+  private final Measure measure;
+  /**
+   * The column name.
+   * <p>
+   * This is the name of the column, and should be unique in a list of columns.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final ColumnName name;
+  /**
+   * The pricing rules that apply to this column, overriding the default rules.
+   * In most cases, there is no need to override the rules on a per-column basis.
+   */
   @PropertyDefinition(validate = "notNull")
   private final PricingRules pricingRules;
-
-  /** The market data rules that apply to this column in addition to the default rules. */
+  /**
+   * The market data rules that apply to this column, overriding the default rules.
```

Comment:
The previous description was more accurate. The rules are merged with the column rules taking precedence. But if the column rules don't have an answer then the default rules are used. "overriding" doesn't make that as clear as "in addition to".